packages/gatsby/src/utils/webpack.config.js

```diff
@@ -675,6 +680,23 @@ module.exports = async (
 
     config.externals = [
       function (context, request, callback) {
+        if (
+          stage === `develop-html` &&
+          isCI() &&
```

Comment:
This and the above CI check do significantly different things, Also most of the CIs don't have all `CI` env set.

Reply:
Above check (lines 99-102) probably can be discarded now (it was used in previous implementation where `CI` env var was "forwarded" to frontend with webpack's DefinePlugin. Now it's not used anywhere

Reply:
yeah removed 👍